src/EnergyPlus/main.cc

```diff
@@ -63,6 +63,8 @@ using EnergyPlus::CommandLineInterface::ProcessArgs;
 int
 main( int argc, const char * argv[] )
 {
+	// the following line is only needed when debugging issues related to NaN in Visual Studio. See https://github.com/NREL/EnergyPlus/wiki/Debugging-Tips
+	// unsigned int fp_control_state = _controlfp( _EM_INEXACT, _MCW_EM );
```

Comment:
+1 for ifdef 

Are you wanting it to always be built on Visual Studio builds in debug mode?  If so, then I'd wrap it locally with an `#ifdef` for that particular combination.  Visual Studio defines `_DEBUG` for debug builds, and you should be able to use the `MSC_VER` flag or something like that to check if it is Visual Studio.

If you are just wanting to enable it sometimes, I would create a new `#define` at the top of this file with some descriptive name like `#define VISUAL_STUDIO_NAN_DEBUG`, and wrap it with that `#ifdef`.

I wouldn't add anything to the CMake build setup.